### PR TITLE
OSDOCS#7094: Added warning for MTU value

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OSDOCS-7094

Link to docs preview: https://62962--docspreview.netlify.app/openshift-enterprise/latest/networking/hardware_networks/configuring-sriov-device.html#nw-sriov-networknodepolicy-object_configuring-sriov-device

QE review:
- [x] QE has approved this change. @zhaozhanqi 
- [x] SME has approved this change.

### CODE REVIEW

**Comment by zhaozhanqi** [comment]
this should be only for customer want to create VFs on PF which is the default network interface I think  cc @bn222 @wizhaoredhat  correct me if not.

File: modules/nw-sriov-networknodepolicy-object.adoc [hunk 1]
```diff
@@ -50,6 +50,11 @@ When specifying a name, be sure to use the accepted syntax expression `^[a-zA-Z0
 <5> Optional: The priority is an integer value between `0` and `99`. A smaller value receives higher priority. For example, a priority of `10` is a higher priority than `99`. The default value is `99`.
 
 <6> Optional: The maximum transmission unit (MTU) of the virtual function. The maximum MTU value can vary for different network interface controller (NIC) models.
++
+[IMPORTANT]
+====
+Ensure that the MTU is not set to a value that does not match the cluster MTU.
```
Comment (zhaozhanqi) [inline modules/nw-sriov-networknodepolicy-object.adoc:56]:
```suggestion
Ensure that the MTU is not set to a value that does not match the cluster MTU if you want to create VFs on default network interface
```

**Comment by zhaozhanqi** [comment]
> @zhaozhanqi This is correct. The PF would be the default OVN-K network interface for this issue to cause node connectivity problems.

Thanks @wizhaoredhat for the confirm. 
So I suggest we can update to `Ensure that the MTU is not set to a value that does not match the cluster MTU if you want to create VFs on default network interface`

File: modules/nw-sriov-networknodepolicy-object.adoc (hunk 1 above)
Comment (snarayan-redhat) [reply modules/nw-sriov-networknodepolicy-object.adoc]: @zhaozhanqi - Can you please check if the fix looks fine now.

**Comment by wizhaoredhat** [comment]
LGTM

**Comment by dfitzmau** [comment]
Hi @snarayan-redhat . LGTM, but I'll need @mburke5678 to approve my comment as per OCP training requirements.

**Comment by mburke5678** [comment]
@snarayan-redhat @dfitzmau LGTM. Don't forget to squash your commits.

**Pull request merged**